airbyte-ci/connectors/pipelines/pipelines/actions/environments.py

```diff
@@ -484,9 +485,30 @@ def with_bound_docker_host(
     )
 
 
+def with_bound_custom_docker_host(
+    context: ConnectorContext,
+    container: Container,
+    custom_docker_host_name: str,
+) -> Container:
+    """Bind a container to a docker host. It will use the dockerd service as a docker host.
+
+    Args:
+        context (ConnectorContext): The current connector context.
+        container (Container): The container to bind to the docker host.
+    Returns:
+        Container: The container bound to the docker host.
+    """
+    dockerd = with_dockerd_service(context.dagger_client)
+    return (
+        container.with_env_variable("DOCKER_HOST", f"tcp://{custom_docker_host_name}:2375")
+        .with_service_binding(custom_docker_host_name, dockerd)
+        .with_mounted_cache("/tmp", context.dagger_client.cache_volume("shared-tmp"))
+    )
+
+
 def bound_docker_host(context: ConnectorContext) -> Container:
     def bound_docker_host_inner(container: Container) -> Container:
-        return with_bound_docker_host(context, container)
+        return with_bound_global_docker_host(context, container)
```

Comment:
How come we are not passing in a value for `custom_docker_host_name` here?

Reply:
Because `with_bound_global_docker_host` has a static hostname: `global-docker-host`. The `custom_docker_host_name` parameter is only for `with_bound_custom_docker_host` function. I'll try to make a change to make it more obvious.